crates/astria-sequencer-relayer/src/relayer.rs

```diff
@@ -117,8 +117,8 @@ impl Relayer {
             sequencer_poll_period: Duration::from_millis(cfg.block_time),
             data_availability,
             // FIXME (https://github.com/astriaorg/astria/issues/509): allow configuring this
```

Comment:
should we address #509 at some point? how does the default pricing get determined (some oracle in the celestia node?)

Reply:
Celestia node contains code to estimate the gas fee programatically: https://docs.celestia.org/developers/submit-data#estimating-gas-programmatically

At some point it would be good to have an option here to set manually, or use default.